Updates to activity view

jordan-ae opened the pull request:
## Describe your changes
https://www.loom.com/share/ada5e5379ade4efb912ed9dabf4a63dc?sid=ca9ea714-8174-4b1f-92e2-5403b2aa9cb8
![Screenshot from 2025-02-15 14-21-32](https://github.com/user-attachments/assets/498e48d9-0180-4fd5-9092-b72abf8a8e54)

## Issue ticket number and link
https://community.sphinx.chat/p/csvm3cqtu2rphloa7870/assigned/3758/0

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update


## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

humansinstitute commented:
@jordan can you update for the conflicts, this is likely to include the new eft hand nav Ahsan has implemented. 

Also the "Add Activity" button on the left hand side, should not be needed.

jordan-ae commented:
@humansinstitute The sidebar implementation is really poorly done. First off all the sidebar component is not a standalone component it's just thrown into the hive chat activity component. Component can't be reusable. 

   - The sidebar should take up ~30% of the screen when open, but currently, it covers the already existing data on the page making it feel broke
   - It’s always open by default, but the layout doesn’t accommodate this properly.
   - When collapsed, all the feature icons look identical, making it unclear what each option represents.
   
   I've removed the activity button and fixed the merge conflicts
   
![Screenshot from 2025-02-16 13-02-44](https://github.com/user-attachments/assets/9e815459-3483-4159-8553-66db576ffd77)

PS: Screenshot was taken before fix that's why the activity button is still ther
![Screenshot from 2025-02-16 12-55-38](https://github.com/user-attachments/assets/e43b3b9b-7327-4af4-b974-1b9ab5383928)
